lib/uplink/ext/common_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/pkg/storj"
+)
+
+func TestCTests(t *testing.T) {
+	ctests, err := filepath.Glob(filepath.Join(CTestsDir, "*_test.c"))
+	require.NoError(t, err)
+
+	t.Run("all", func(t *testing.T) {
+		for _, ctest := range ctests {
+			t.Run(ctest, func(t *testing.T) {
+				t.Parallel()
```

Comment:
btw, for some reason, some tests are not failing when `t.Parallel()` is used here. not sure why yet
when its disabled:
```
            test_utils.go:106: /home/wywrzal/git/storj/lib/uplink/ext/c/tests/object_test.c:74:TestObject:FAIL: Expected 'testing data 123' Was 'testing data 123\x10'
                
                -----------------------
                1 Tests 1 Failures 0 Ignored 
                FAIL
                
            require.go:794: 
                	Error Trace:	test_utils.go:107
                	            				test_utils.go:124
                	            				common_test.go:42
                	Error:      	Received unexpected error:
                	            	exit status 1
                	Test:       	TestCTests/all//home/wywrzal/git/storj/lib/uplink/ext/c/tests/object_test.c
FAIL
FAIL	storj.io/storj/lib/uplink/ext	32.815s
lib/uplink/ext/tests.go:5: running "go": exit status 1
```